server/utils/appendFieldUtils.ts

```diff
@@ -0,0 +1,26 @@
+import { Request } from 'express'
+import { formatDisplayDateTime } from './datetimeUtils'
+import { lastNameCommaFirstName } from './miniProfileUtils'
+import Prisoner from '../services/prisonerSearch/prisoner'
+
+/**
+ * If the description + divider test is equal to or more than 3000 characters, then show the character count
+ * If the description + divider text is less than 3000 characters, do not show the character count until the user reaches a total of 3000 characters for all entries plus dividers
+ */
+export const generateSaveTimestamp = (prisoner: Pick<Prisoner, 'firstName' | 'lastName'>) =>
+  `\n\n[On ${formatDisplayDateTime(new Date().toString())}, ${lastNameCommaFirstName(prisoner)} added:]\n\n`
```

Comment:
Design has this as "lastname firstname" not "firstname, lastname" unless that's changed post ticketing.

Reply:
![image](https://github.com/user-attachments/assets/72056f8d-5aa7-42a8-a49b-ea5bfa05bc00)

Reply:
I can clarify with Matt if this is clashing with the designs though

Reply:
I assume this should be `res.locals.user.displayName.substring(0, 240)` as per `area-of-work` assuming designs are correct?

Reply:
Just `res.locals.user.displayName`. No need to truncate it as the other length checks will account for it.